engine/runtime/src/secret.rs

```diff
@@ -185,13 +185,11 @@ mod tests {
 
     use super::*;
 
-    use std::{format, ptr};
+    use std::format;
 
-    use libsodium_sys::randombytes_buf;
-
-    pub(crate) fn memrandom(bytes: &mut [u8]) {
-        unsafe { randombytes_buf(bytes.as_mut_ptr() as *mut _, bytes.len()) }
-    }
+    // pub(crate) fn memrandom(bytes: &mut [u8]) {
```

Comment:
why did we need to remove this?

Reply:
Because for whatever reason the github virtual machines didn't let the hack work as it does on a local machine